### ISSUE TITLE
intelliJ 18.3.6 incomaptibility

### ISSUE THREAD

**Issue by la10736** [intellij-rust/intellij-rust#3607]
## Environment

* **Intellij-Rust plugin version:** 0.2.95.2120-183
* **Rust toolchain version:** 1.32
* **IDE name and version:** IntelliJ 2018.3.6 Community
* **Operating system:** Ubuntu 16.04

## Problem description
Last Intellj 2018.3.6 seams to be incompatible with plug-in

## Steps to reproduce

After last Intellj update to 2018.3.6 versions and rust plug-in (as suggested by the IDE). Is stop to works and update fails due plug-in  incompatibility

Rollback to 2018.3.5 _fix the issue_

**Comment by Undin** [comment]
@la10736 Unfortunately, we can't reproduce this issue
Could you reinstall plugin?

**Comment by la10736** [comment]
I already did it: after removed it when I try to install intellJ say that is incompatible.

Unfortunately I cannot post image from work office (maybe github use some sort of aws-lambda service...). I come back later with the image.

**Comment by la10736** [comment]
Maybe something is break on idea side (maybe on the patch system): today I did the upgrade to 2019.1 and that brake my installation. Clean up configurations didn't fix... I solved just by download the version from scratch and install it.

After this experience I downloaded 2018.3.6 version and installed from scratch, just take the configurations. Now install plug-in works.

Now I guess that that's not an intellij-rust's bug but an ItelliJ one on the patch system.

Feel free to close it.
